ansible/roles/pmm2-images/tasks/main.yml

```diff
@@ -197,17 +179,28 @@
       with_items:
         - /srv/clickhouse
 
-    - name: Grafana                   | Enable gzip for grafana
-      ini_file:
-        dest: /etc/grafana/grafana.ini
-        section: server
-        option: enable_gzip
-        value: "true"
-
+    - name: PMM update                | Create pmm-update directory
+      file:
+        path: /srv/update
+        state: directory
+ 
     - name: Create tmp dirs           | Create tmp dirs
       when: ansible_virtualization_type != "docker"
       command: /usr/bin/systemd-tmpfiles --create --remove --boot --exclude-prefix=/dev
 
+    - name: Clickhouse logger         | Configure clickhouse logger
+      replace:
+        path: /etc/clickhouse-server/config.xml
+        regexp: '(<logger>[\s\S]*)</logger>'
+        replace: |-
+          <logger>
+                  <!-- Possible levels: https://github.com/pocoproject/poco/blob/develop/Foundation/include/Poco/Logger.h#L105 -->
+                  <level>trace</level>
+                  <console>1</console>
+              </logger>
```

Comment:
This misalignment is on purpose )

Reply:
Question, why is this task duplicated from https://github.com/percona/pmm-update/pull/128/ ?
Can't you avoid the duplcation?

Reply:
It's not a duplicate, it's the way it was designed a while back.